source/blog/paginated.html.erb

```diff
@@ -4,18 +4,36 @@ section_class: 'blog_paginated section_blog'
 layout: blog_category
 include_campaign_monitor: true
 ---
-<% locale_obj = locals[:locale_obj] || current_locale_obj %>
+<%
+  locale_obj = locals[:locale_obj] || current_locale_obj
+  is_all_category = category[:name] == 'All'
+  is_top_20_category = sharesight_top_20_category && category[:id] == sharesight_top_20_category[:id]
+  show_menu = !is_top_20_category
+  posts_per_row = show_menu ? 2 : 3
+%>
 
 <div class="blog">
   <header class="blog__header">
     <div class="container">
 
-      <a href="<%= unlocalized_url('blog') %>" class="breadcrumb" title="Sharesight Blog"><% if category[:name] != 'All' %>Blog<% end %></a>
+      <% unless is_all_category %>
+        <a href="<%= unlocalized_url('blog') %>" class="breadcrumb" title="Sharesight Blog">Blog</a>
+      <% end %>
+
+      <% if is_all_category %>
+        <a href="<%= unlocalized_url('blog') %>" title="Sharesight Blog"><h1 class="heading_page">Blog</h1></a>
+      <% elsif is_top_20_category %>
+        <h1 class="heading_page"><%= space_category_page_title %></h1>
 
-      <% if category[:name] != 'All' %>
-        <h1 class="heading heading_page heading_page_index">Topic: <%= space_category_page_title %></h1>
+        <div class="campaign_monitor">
+          <%#
+            Campaign Monitor "Subscribe" Button.
+            TODO: This should be specifically for the Sharesight Top 20
```

Comment:
What's the `TODO` for? Isn't it now already Sharesight Top 20 only?

Reply:
No, need content from marketing.

Reply:
I see.